src/About.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <body>
+        <div>
+          <div>Login</div>
```

Comment:
Rather than duplicate code, try to create a NavBar Component with the three elements inside.
This way you only have to update the navbar component once rather than in each file.